### PR TITLE
php 8.3 support

### PR DESCRIPTION
**Pull request by krumynin1993**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "description": "Doctrine Cache adapter for PSR-16 Simple Cache",
     "type": "library",
     "require": {
-        "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
+        "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0",
```
Comment (Ocramius) [inline composer.json:6]: `composer.lock` hash needs appropriate update too.

**Review by Ocramius — CHANGES_REQUESTED** [review]
See also #59

File: composer.lock [hunk 2]
```diff
@@ -376,16 +376,16 @@
         },
         {
             "name": "composer/pcre",
-            "version": "3.1.0",
+            "version": "3.1.1",
```
Comment (Ocramius) [inline composer.lock]: Uhh, upgrading everything may cause quite a mess. See #57

Comment (krumynin1993) [reply composer.lock]: done, updated only php

**Comment by krumynin1993** [comment]
hello, can you merge it please and make a new release ?)

**Review by Ocramius — APPROVED** [review]
Yup, thanks @krumynin!

**Pull request merged**